01/assets/js/app.js

```diff
@@ -1,4 +1,44 @@
-const a = '4.2';
-const b = 9;
+const a = '4.2'
+const b = 9
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b)
+console.log(typeof a)
+console.log(typeof b)
+
+const sum = parseFloat(a) + b
+const subtraction = parseFloat(a) - b
+const multiply = parseFloat(a) * b
+const divide = parseFloat(a) / b
+const modulo = parseFloat(a) % b
+
+console.log(sum)
+console.log(subtraction)
+console.log(multiply)
+console.log(divide)
+console.log(modulo)
+
+if (sum > 20) {
+	console.log('jest większy niż 20')
+} else {
+	console.log('nie jest większy')
+}
+if (subtraction > 20) {
+	console.log('jest większy niż 20')
+} else {
+	console.log('nie jest większy')
+}
+if (multiply > 20) {
+	console.log('jest większy niż 20')
+} else {
+	console.log('nie jest większy')
```

Comment:
Zobacz, że tutaj powtarzasz to samo działanie.
Można by było dodać wyniki działania do tablicy i wykorzystać pętle tj.
```
const results = [];
results.push(sum);
results.push(multiply)
// ...

for(let i=0; i<results.length; i++) {
    if(results[i] > 20) {
        // ...
    } else {
        // ...
    }
}
```